docs/personal_data_handling.md

```diff
@@ -0,0 +1,56 @@
+# Data handling in PMM - FAQ
+
+
+The following questions are being answered related to personal and confidential data handling in PMM:
+
+1. Which type of data is transmitted?
+
+      |**Data collection source**                                       | **Data collected** |
+      | --------------------------------------------------------------- | ------------------------------------------------------
+      | DB host to PMM                                                  | Database performance metrics <br/> SQL query examples for query analytics (optional).
+      | PMM to DB Host                                                  | DSN and credentials for database access. A separate DB user is used (read-only) to access the database.
```

Comment:
Is this true?  I seem to recall you need some amount of privs to flush/rotate either perf schema or slowlog... @rnovikovP am I off here?  I think "limited access account" might be a better fit if so.

Reply:
here DSN and credentials are referred to "limited access account" (pmm), but anyway they are passed to pmm-agent so that it is able to connect to DB.

Reply:
@shoffman-percona, it's not, we don't create a separate DB user. So we use credentials passed by the user.

Reply:
Yes but the idea is that PMM collects and stores this data (plain text) in it's master database (pg) so we need to disclose that to the users...my question was more about the privs required for "best practice" configuration...is it truly a "read-only" user we ask them to create or does that user have some ability to modify a remote system?

Reply:
it's not read-only. as you can see in following links we request more permissions then just read.
https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mongodb.html#create-roles-with-privileges-for-backups-and-qan
https://docs.percona.com/percona-monitoring-and-management/setting-up/client/postgresql.html#create-a-database-account-for-pmm
https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mysql.html#create-a-database-account-for-pmm
I would recommend not mentioning it here at all. Because it seems like we create a separate DB user.

Reply:
perfect...we'll remove the read-only part but we do need to disclose that we are capturing and storing this user-created db users credentials in PMM along with DSN.